tools/run_blob_bench.sh

```diff
@@ -74,6 +75,11 @@ if [ $# -ge 1 ]; then
   fi
 fi
 
+if [ ! -f tools/benchmark.sh ]; then
```

Comment:
Thanks for adding the path check! @BilyZ98   Is this script path check needed though, given that it's part of the repo, in what scenario are you thinking this script could be missing?

Reply:
I thought it was something like the working directory is tools/ and the user runs ./run_blob_bench.sh ; then tools/benchmark.sh would not be found because its relative path is just ./benchmark.sh